src/core/server/plugins/legacy_config.ts

```diff
@@ -60,3 +70,33 @@ export const getGlobalConfig$ = (configService: IConfigService): Observable<Shar
     )
   );
 };
+
+// TODO: move this to Elasticsearch service itself
```

Comment:
Why not expose `ssl. certificateAuthorityFingerprints` thing from elasticsearch `setup` contract?
```typescript
interface InternalElasticsearchServiceSetup {
  readonly config$: Observable<{
    readonly ssl: {
      readonly certificateAuthorityFingerprints: readonly string[];
    }
  }>
}
```
We are going to get rid of `legacy config` anyway